components/templates/template.html.twig

```diff
@@ -560,7 +571,13 @@
                                           :formatter="column.formatter">
                         </datatable-column>
                         <template slot="actions_materials" scope="cell">
-                            <div v-if="cell.row['is_active']=='1'"><span @click="deleteResource('materials', cell)"><i class="fa fa-trash-o 2x" aria-hidden="true"></i></span></div>
+                            <div v-if="cell.row['is_active']=='1'"  class="right-distance">
```

Comment:
Do you need `v-if="cell.row['is_active']=='1'" ` ? I introduced this for the other datatable view, because admins can delete (== set `cell.row['is_active']=0`) Resources. Translators can **NOT** delete anything, so we should never need to double check this value. Please remove if not needed